components/voice_assistant.rst

```diff
@@ -82,6 +82,19 @@ Configuration:
 - **volume_multiplier** (*Optional*, float): Volume multiplier to apply to the assist pipeline.
   Must be larger than 0. Defaults to 1 (disabled).
 
+- **on_timer_started** (*Optional*, :ref:`Automation <automation>`): An automation to perform when a voice assistant
+  timer has started. The timer is available as ``timer`` of type :apistruct:`voice_assistant::Timer`.
+- **on_timer_finished** (*Optional*, :ref:`Automation <automation>`): An automation to perform when a voice assistant
+  timer has finished. The timer is available as ``timer`` of type :apistruct:`voice_assistant::Timer`.
+- **on_timer_cancelled** (*Optional*, :ref:`Automation <automation>`): An automation to perform when a voice assistant
+  timer has been cancelled. The timer is available as ``timer`` of type :apistruct:`voice_assistant::Timer`.
+- **on_timer_updated** (*Optional*, :ref:`Automation <automation>`): An automation to perform when a voice assistant
+  timer has been updated. The timer is available as ``timer`` of type :apistruct:`voice_assistant::Timer`.
```

Comment:
This is actually only called when the timer is paused and resumed. Update your suggestion for this

Reply:
Make it a committable suggestion

Reply:
I think he means make it a GH suggestion (not a diff), like this:

```suggestion
- **on_timer_updated** (*Optional*, :ref:`Automation <automation>`): An automation to perform when a voice assistant
  timer has been paused and/or resumed. The timer object, of type :apistruct:`voice_assistant::Timer`, is available as ``timer``.
```

...because then you can commit it directly from GH and don't have to commit the patch locally. It's just easier.

Reply:
Ho does this bot work? lol

Reply:
I do think it would be better to expand on "updated". 

```suggestion
- **on_timer_updated** (*Optional*, :ref:`Automation <automation>`): An automation to perform when a voice assistant
  timer has been updated (paused/resumed/duration changed). The timer is available as ``timer`` of type :apistruct:`voice_assistant::Timer`.
```